transport/peer/peerlist/single.go

```diff
@@ -0,0 +1,76 @@
+package peerlist
+
+import (
+	"context"
+
+	"sync"
+
+	"go.uber.org/yarpc/transport"
+	"go.uber.org/yarpc/transport/internal/errors"
+)
+
+type single struct {
+	lock sync.RWMutex
+
+	initialPeerID transport.PeerIdentifier
+	peer          transport.Peer
+	agent         transport.Agent
+	started       bool
+}
+
+// NewSingle creates a static PeerList with a single Peer
+func NewSingle(pi transport.PeerIdentifier, agent transport.Agent) transport.PeerList {
```

Comment:
![85351-dale-shhh-gif-the-walking-dead-pzrw](https://cloud.githubusercontent.com/assets/613317/20165560/3ef6e29a-a6c3-11e6-8a2d-7a3709e07a74.gif)